Resources/Prototypes/Corvax/Recipes/Reactions/drinks.yml

```
@@ -0,0 +1,159 @@
+    Sugar:
+      amount: 1
+    JuiceLime:
+      amount: 1
+    Gin:
+      amount: 3
+  products:
+    Bramble: 6
+
+- type: reaction
+  id: MaiTai
+  reactants: # Orgeat may be added
```

yamllint

[warning] too few spaces before comment

(comments)